pkgs/by-name/we/webcamize/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  makeWrapper,
+  ffmpeg,
+  libgphoto2,
+  kmod,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
You can look for the `finalAttrs` version, which is prefered inside nixpkgs for performance reasons